hudi-common/src/main/java/org/apache/hudi/common/model/HoodieBaseFile.java

```diff
@@ -52,14 +61,45 @@ public HoodieBaseFile(String filePath) {
   public HoodieBaseFile(String filePath, BaseFile bootstrapBaseFile) {
     super(filePath);
     this.bootstrapBaseFile = Option.ofNullable(bootstrapBaseFile);
+    String[] fileIdAndCommitTime = getFileIdAndCommitTimeFromFileName();
+    this.fileId = fileIdAndCommitTime[0];
+    this.commitTime = fileIdAndCommitTime[1];
+  }
+
+  /**
+   * Parses the file ID and commit time from the fileName.
+   * @return String array of size 2 with fileId as the first and commitTime as the second element.
+   */
+  private String[] getFileIdAndCommitTimeFromFileName() {
```

Comment:
can we have this method in FSUtils only and call it from here.

Reply:
let's do that when needed.